### PR TITLE
#48 add unit test for RqMethod

### PR DESCRIPTION
**Pull request by dmzaytsev**

### CODE REVIEW

**Comment by davvd** [comment]
Thanks, let me find someone who can review this pull request

**Comment by davvd** [comment]
@pinaf please review, thanks

File: src/test/java/org/takes/rq/RqMethodTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+/**
+ * Test case for {@link org.takes.rq.RqMethod}.
+ * @author Dmitry Zaytsev (dmitry.zaytsev@gmail.com)
+ * @version $Id$
+ * @since 0.9.1
+ */
+public final class RqMethodTest {
+
+    /**
+     * RqMethod can get a method.
```
Comment (pinaf) [inline src/test/java/org/takes/rq/RqMethodTest.java]: @dmzaytsev How about **can return its method**? Sound better?

File: src/test/java/org/takes/rq/RqMethodTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void getMethod() throws IOException {
```
Comment (pinaf) [inline src/test/java/org/takes/rq/RqMethodTest.java]: @dmzaytsev `returnsMethod()`

**Comment by pinaf** [comment]
@dmzaytsev just method/javadoc naming. thanks.

**Comment by dmzaytsev** [comment]
@pinaf please review again, thanks

**Comment by pinaf** [comment]
@dmzaytsev looks good. thanks!

**Comment by pinaf** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge pls

**Comment by pinaf** [comment]
@yegor256 network failure.

**Comment by yegor256** [comment]
@rultor try to merge again

**Comment by davvd** [comment]
@pinaf **16 mins** added to your account (payment number `AP-8N773154Y04855251`), many thanks for your contribution!; review comments (c=1) added as a bonus; +16 added to your rating, at the moment it is: [+5016](http://www.netbout.com/b/35128?open=rating)

**Comment by davvd** [comment]
@rultor deploy now pls

**Pull request merged**